### PR TITLE
simplify the demo setup

### LINKED ISSUE #29: simplify the demo setup
For historical reasons Broadway shipped with MySQL and Elasticsearch out of the box. This makes it a little difficult to set up. Now we have multiple event store and read model implementations. 

I suggest to refactor this demo to use the simplest backing services as possible and in general simplify the project to get the demo up and running in no time for new users.

### PR DESCRIPTION
**Pull request by othillo**
fixes #29 #31
depends https://github.com/broadway/coding-standard/pull/1

- [x] use SQLite to simplify the infrastructure (drop Elasticsearch, MongoDB, Ansible, docker)
- [x] use Symfony MicroKernelTrait to simplify the or even remove the bundle
- [x] use lastest Broadway bundle
- [x] improve documentation
- [x] use Symfony 4
- [x] improve read model creation
- [x] add event store commands (https://github.com/broadway/broadway-bundle/commit/5cc22d5dce4f498c96886f83be53e7838e3130cc)
- [x] apply coding standards
- [x] fix license docblocks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,88 +3,85 @@
...
 ```
-
-Docker-compose will set up the containers needed to run this demo.
-
-
-```sh
-docker-compose exec app /bin/bash
-app/console broadway:event-store:schema:init
+composer install
+bin/console broadway:event-store:schema:init
```
Comment (wjzijderveld) [inline README.md]: `bin/console doctrine:database:create`?
But there is no way to create the schema right now... :crying_cat_face:

File: public/index.php [hunk 2]
```diff
@@ -17,10 +17,7 @@
 
 require __DIR__ . '/../app/AppKernel.php';
 
-$debug       = true;
-$environment = 'dev';
-
-$kernel = new AppKernel($environment, $debug);
+$kernel = new AppKernel('dev', true);
```
Comment (wjzijderveld) [inline public/index.php]: This index.php hasn't been updated with the kernel changes (namespace, sf4 kernel)

File: README.md [hunk 3]
```diff
@@ -3,88 +3,85 @@
...
-```sh
-cd /vagrant
-phpunit --exclude-group=none
 ```
-
-### Docker Compose
-
-```sh
-docker-compose run app vendor/bin/phpunit --exclude-group=none
+./vendor/bin/phpunit --exclude-group=none
```
Comment (wjzijderveld) [inline README.md]: As we simplified it now anyway, let's change it to run everything be default?

File: symfony.lock [hunk 4]
```diff
@@ -0,0 +1,212 @@
+{
```
Comment (wjzijderveld) [inline symfony.lock]: After installation this lockfile already had changes. Can we fix that?
(symfony/symfony and symfony/polyfill-ctype were added).

File: README.md (hunk 1 above)
Comment (othillo) [reply README.md]: there are commands in this repo: https://github.com/othillo/broadway-demo/blob/49b8139a35d47ecd61fb0e5c599dfc5222f7deae/src/Command/CreateReadModelCommand.php#L41

File: symfony.lock (hunk 4 above)
Comment (othillo) [reply symfony.lock]: I added the updated symfony.lock

File: public/index.php (hunk 2 above)
Comment (othillo) [reply public/index.php]: I imported the namespace. What else is missing?

File: README.md (hunk 1 above)
Comment (wjzijderveld) [reply README.md]: But only for the readmodels, not for the eventstore?

I noticed after my comment that the doctrine command only creates the sqlite file, nothing else. So pretty pointless :D

File: public/index.php (hunk 2 above)
Comment (wjzijderveld) [reply public/index.php]: I removed the call to `loadClassCache`, that at least made it work, but I haven't checked if there is al alternative 🙈

File: README.md (hunk 1 above)
Comment (othillo) [reply README.md]: see https://github.com/broadway/broadway-bundle/issues/54#issuecomment-397655283

**Pull request closed without merging**